web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx

```diff
@@ -161,6 +187,22 @@ export const ChangesetNode: React.FunctionComponent<ChangesetNodeProps> = ({
                 )}
             </div>
             <div className="flex-shrink-0 flex-grow-0 ml-1 d-flex flex-column align-items-end">
+                {node.__typename === 'ExternalChangeset' && (
+                    <span
+                        data-tooltip={
+                            node.updatedAt === lastUpdatedAt
+                                ? 'Currently refreshing'
+                                : `Last synced ${formatDistanceToNow(
+                                      parseISO(node.updatedAt)
```

Comment:
Isn't `new Date()` sufficient?
```suggestion
                                      new Date(node.updatedAt)
```

Reply:
it only accepts proper ISO format and doesn't respect local differences like

`03-02-2020`
US: March 3, 2020
World: 3. February 2020

But it's not important here :)